packages/pong-engine/src/state.ts

```diff
@@ -62,21 +64,22 @@ export const reducer = (
 
   switch (type) {
     case 'move':
-      const { deltaTime } = payload;
       return {
         ...state,
-        ball: getBallPos(ball, paddle, deltaTime),
-        paddle: getPaddlePos(paddle, deltaTime),
+        ball: getBallPos(ball, paddle, payload.deltaTime),
```

Comment:
why don't we like destructuring the object here but we like it in L101?

Reply:
Because we are also using deltaTime inside other case condition and, since the scope of the switch statement is the same for all case conditions, we cannot declare twice the variable